mongoid.gemspec

```diff
@@ -32,7 +32,7 @@ Gem::Specification.new do |s|
   s.required_ruby_version     = ">= 2.3"
   s.required_rubygems_version = ">= 1.3.6"
 
-  s.add_dependency("activemodel", [">= 5.1", "<6.1"])
+  s.add_dependency("activemodel", [">= 5.1", "<=6.1"])
```

Comment:
I would suggest making it "< 6.2", because if it is compatible with 6.1, it will most likely be compatible with 6.1.1 and so on.

Reply:
OK. thanks.